pdc/apps/auth/models.py

```diff
@@ -82,3 +82,7 @@ def email_user(self, subject, message, from_email=None, **kwargs):
         Sends an email to this User.
         """
         send_mail(subject, message, from_email, [self.email], **kwargs)
+
+    def get_all_permissions(self, obj=None):
+        tmp_permission = _user_get_all_permissions(self, obj)
+        return sorted(tmp_permission)
```

Comment:
_user_get_all_permissions looks like it want to be a private method. Is it appropriate to use here? Is there get_all_permissions in parent class we can call it to get result and sort?

BTW: this 2 line also could be
```
return sorted(_user_get_all_permissions(self, obj))
```